autoload/lsp/ui/vim/output.vim

```diff
@@ -173,6 +173,30 @@ function! lsp#ui#vim#output#adjust_float_placement(bufferlines, maxwidth) abort
     endif
 endfunction
 
+function! lsp#ui#vim#output#popup_scroll(val) abort
+    let l:pos = popup_getpos(s:winid)
```

Comment:
this will fail for neovim.

can we also have support for neovim?

Reply:
Yep, I managed to work it out. The implementation is perhaps not as nice as I would like, but I don't know if there's a better way.